avpaul/politico
# #164101104 Add new end points, Fix errors and add new test cases

#### What does this PR do?
- Add reset password API endpoint
- Add get results API endpoint
- Fix Travis Postgresql database creation error
- Add new test cases

#### Description of Task to be completed?
- User should be able to get a link to reset his/her password

#### How should this be manually tested?
- Check the Travis CI build
- Test it locally: you need Git, Postgresql, NodeJs/NPM and Postman installed
  ```JSON
    psql createdb politico_test
    Git clone repo_url && cd politico-db
    npm install
    npm run createdb && npm run createAdmin //For creating the database and the admin
    npm test
  ```

#### What are the relevant pivotal tracker stories?
#164101104  #1640556562 #164058651
